rdkit/Chem/Descriptors.py

```diff
@@ -88,6 +88,39 @@ def _setupDescriptors(namespace):
   31.05...
 
 """
+def GetNumChiralCenters(mol):
+  """The number of chiral centers in a molecule
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(N)O'))
+  1
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(Br)C(N)O'))
+  2
+  
+  """
+  chiralcenters = Chem.FindMolChiralCenters(mol, includeUnassigned=True)
+  return len(chiralcenters)
+
+
+def GetNumSpecifiedChiralCenters(mol):
+  """Number of specified chiral centers in a molecule
+  >>>>  GetNumSpecifiedChiralCenters(Chem.MolFromSmiles('C[C@H](N)O'))
+  1
+  
+  """
+  specificcenters = Chem.FindMolChiralCenters(mol)
+  return len(specificcenters)
+
+def GetNumUnspecifiedChiralCenters(mol):
+  """Number of unspecified chiral centers in a molecule
+  >>>> GetNumUnspecifiedChiralCenters(Chem.MolFromSmiles('CC(N)O'))
+  1
+  
+  """
+  unspeccenters = 0
+  unspecificcenters = Chem.FindMolChiralCenters(mol, includeUnassigned=True)
+  for center in unspecificcenters:
+    if center[1] == '?':
+      unspeccenters += 1
+  return unspeccenters
```

Comment:
I see you fixed the error. 

You can also make this a list comprehension:
```
matches = sum(1 for x in Chem.FindMolChiralCenters(mol, includeUnassigned=True) 
              if x[1] == '?')
```
or make use of the other two functions. A bit slower though.
```
matches = GetNumChiralCenters(mol) - GetNumSpecifiedChiralCenters(mol)
```